### DISCUSSION TITLE
Problem connecting to canvas : AttributeError: 'NoneType' object has no attribute 'mapSettings'

### DISCUSSION THREAD

**Discussion by caslumali** [EnMAP-Box/enmap-box#689] [Q&A]
Hello everybody,

I installed the application and all Python packages and it loads without any problems. However, I can't use it because I think there is a connection error between Enmapbox and the raster displayed on the canvas.

When I click "apply" in any of the enmapbox tolls this is the Erreur Python that appears:

python\plugins\enmapboxplugin\enmapbox\coreapps\scatterplotapp\scatterplotdialog.py", line 192, in currentExtent
              mapSettings: QgsMapSettings = self.mMapCanvas.mapSettings()
             AttributeError: 'NoneType' object has no attribute 'mapSettings'

I'm using the latest LTR version of QGIS: 3.28.11 on Windows 10

Best Regards

**Comment by janzandr** [comment]
Hi @caslumali,
I can't reproduce this.
1. make sure that you restarted QGIS after installation of the EnMAP-Box.
2. Let's use the scatter plot example you showed above. Show some screenshot or animated GIF and explain which steps you do for reproducing the problem.
